### PR TITLE
fix(encoding): remove encoding for base64 encoded request

### PR DESCRIPTION
**Pull request by fzipi**
Signed-off-by: Felipe Zipitria <felipe.zipitria@owasp.org>

### CODE REVIEW

**Comment by fzipi** [comment]
Encoded request in test has multiple `\\`:
```
R0VUIC8gSFRUUC8xLjFcclxuSG9zdDogZXhhbXBsZS5jb21cclxuXHJcbg== 
```
Gets you `GET / HTTP/1.1\\r\\nHost: example.com\\r\\n\\r\\n`.

File: test/integration/test_http.py [hunk 1]
```diff
@@ -229,11 +228,11 @@ def test4():
 
 
 def test5():
-    """Basic GET without Host on 0.9 - Expect 505 version not supported"""
+    """Basic GET without Host on 0.9- Expect 404"""
     x = ruleset.Input(dest_addr='example.com', version='HTTP/0.9', headers={})
     http_ua = http.HttpUA()
     http_ua.send_request(x)
-    assert http_ua.response_object.status == 505
+    assert http_ua.response_object.status == 404
```
Comment (fgsch) [inline test/integration/test_http.py]: Perhaps we can remove this test altogether. It doesn't look like it's providing much value tbh.

**Review by fgsch — APPROVED** [review]
LGTM

File: test/integration/test_http.py (hunk 1 above)
Comment (fzipi) [reply test/integration/test_http.py]: Test removed

**Pull request merged**